cmd/influxd/server_integration_test.go

```diff
@@ -2020,6 +2020,97 @@ func Test_ServerOpenTSDBIntegration_BadData(t *testing.T) {
 	}
 }
 
+func Test_ServerOpenTSDBIntegrationHTTPSingle(t *testing.T) {
+	if testing.Short() {
+		t.Skip()
+	}
+	nNodes := 1
+	testName := "ServerOpenTSDBIntegrationHTTPSingle"
+	dir := tempfile()
+	now := time.Now().UTC().Round(time.Second)
+	c, _ := main.NewTestConfig()
+	o := main.OpenTSDB{
+		Port:            4245,
```

Comment:
You now need to pass in port 0 to your tests. Please see other tests on why we do this.